tests/e2e/spec/gutenberg_ro.e2e.spec.js

```diff
@@ -349,8 +367,8 @@ function runTests (driver, modes) {
                 // await downloadButton.click();
                 const downloadFileStatus = await driver.wait(async function () {
                     // We can only check if the file exist in firefox and chrome (IE and Edge not supported)
-                    if (!['firefox', 'chrome'].includes(browserName)) {
-                        // will skip if any other browser and pass test
+                    if (!['firefox', 'chrome'].includes(browserName) || BROWSERSTACK) {
```

Comment:
We don't have access to BrowserStack filesystem in CI

Reply:
Ah, OK, makes sense.